### DISCUSSION TITLE
今年のアドベントカレンダーは全て1人で書いた場合の景品はありますか？

### DISCUSSION THREAD

**Discussion by isiskj** [increments/qiita-discussions#920] [ヘルプ]
2022年はありましたが今年は景品ありますか？

**Comment by kimkim0814** [comment]
@isiskj 
お問い合わせありがとうございます。

Xでも告知を行っておりますが、完走賞は今年も実施予定です✨
https://x.com/Qiita/status/1843551939596107938

基準や賞品についての詳細は、11/1（金）オープン予定のイベントページや
[Qiita Advent Calendar 2024 キックオフLIVE](https://increments.connpass.com/event/332583/) での発表をお待ちいただけますと幸いです！

**Reply by isiskj** [reply]
ありがとうございます！
是非参加したいと思います！
